tests/data/RedefinedClassDef.ufo/glyphs/bha-telugu.below.glif

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<glyph name="bha-telugu.below" format="2">
+  <advance width="578"/>
+  <outline>
+  </outline>
+  <lib>
+    <dict>
+      <key>com.schriftgestaltung.Glyphs.category</key>
```

Comment:
if we don't need/use these private lib keys, can you please delete from the test font, otherwise is just noise. It you can make the test font as small as possible (e.g. the smallest required number of glyphs to reproduce) would be best